app/styles/_objects.scss

```diff
@@ -0,0 +1,185 @@
+/**
+    HTML Objects
+    Forms, Inputs, Fields, items, etc
+**/
+
+div[dim-search-filter] {
+    input {
+        width: 250px;
+        margin-top: -3px;
+    }
+}
+
+input, select, option {
+    font-family: 'Open Sans', sans-serif;
+}
+
+/**
+    Icons
+**/
+
+.fa {
+    &:before {
+        font-size: 16px;
+    }
+}
+
+/**
+    Buttons
+**/
+
+.button-name {
+    float: left;
+    width: 123px;
+    background-color: rgba(195, 188, 180, .7);
+    padding-left: 4px;
+    height: 38px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap !important;
+}
+
+.condensed .button-name {
+    width: 178px;
+}
+
+.button-delete, .button-edit {
+    text-align: center;
+}
+
+#faq-button {
+    display: inline-block;
+    background-color: #5b6e6c;
+    color: rgba(245, 245, 245, .6);
+    border-radius: 50%;
+    width: 14px;
+    height: 14px;
+    line-height: 14px!important;
```

Comment:
Good catch, I happened to not write that CSS but I should have caught it nonetheless. That doesn't mess up the CSS, but it renders the `!important` declaration useless.